src/main/resources/anniv5Resources/localization/eng/gemspack/Cardstrings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "${ModID}:TempStrengthGem": {
+    "NAME": "Ruby",
+    "DESCRIPTION": "${ModID}:Gem. NL \"Gain 2 Temp. Strength\"",
+    "UPGRADE_DESCRIPTION": "${ModID}:Gem (2 Uses). NL \"Gain 2 Temp. Strength\""
+  },
+  "${ModID}:WardGem": {
+    "NAME": "Aquamarine",
+    "DESCRIPTION": "${ModID}:Gem. NL \"Gain a *Ward\"",
+    "UPGRADE_DESCRIPTION": "${ModID}:Gem (2 Uses). NL \"Gain a *Ward\""
```

Comment:
What is this 2 uses thing?

Reply:
The socket action downgrades the gem instead of removing it, giving it an extra use.  This is mostly so they have some function for upgrading.  The Guardian version just says "gems can't be upgraded", which I could do as well, it just felt less clean if they had no upgrade.